wc-src/vf-score.js

```diff
@@ -0,0 +1,50 @@
+import Vex from '../src/index.js';
+
+const template = document.createElement('template');
+template.innerHTML = `
+  <div id='vf-score'>
+    <slot></slot>
+  </div>
+`
+
+export class VFScore extends HTMLElement {
+  constructor() {
+    super();
+
+    this.attachShadow({ mode:'open' });
+    this.shadowRoot.appendChild(document.importNode(template.content, true));
+
+    this.addEventListener('getContext', this.getContext);
+    this.addEventListener('getFactory', this.getFactory);
+  }
+
+  connectedCallback() {
+    this.setupVexflow(this.getAttribute('width') || 500, this.getAttribute('height') || 200);
+    this.setupFactory();
+  }
+
+  setupVexflow(width, height) {
+    const div = this.shadowRoot.getElementById('vf-score');
+    const renderer = new Vex.Flow.Renderer(div, Vex.Flow.Renderer.Backends.SVG);
+    renderer.resize(width, height);
+    this.context = renderer.getContext();
+  }
+
+  setupFactory() {
+    this.vf = new Vex.Flow.Factory({renderer: {elementId: null}});
+    this.vf.setContext(this.context);
+  }
+
+  /** Returns the renderer context for this vf-score component */
+  getContext = (e) => {
+    e.detail.context = this.context;
```

Comment:
`CustomEvent.detail` is intended to be readonly per the [spec](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/detail). I suggest setting a property on the child instead. Something like:

```js
e.target.context = this.context;
```

Where `e.target` is the element that dispatched the event.

Then, in the element that dispatches the `getContext` event, it could a `context` property with a setter, something like:

```js
// in the child element 

set context(value) {
  // do something with the value
}
```